llama-index-integrations/llms/llama-index-llms-cohere/llama_index/llms/cohere/utils.py

```diff
@@ -30,9 +33,101 @@
 ALL_AVAILABLE_MODELS = {**COMMAND_MODELS, **GENERATION_MODELS, **REPRESENTATION_MODELS}
 CHAT_MODELS = {**COMMAND_MODELS}
 
+
+def is_cohere_model(llm: BaseLLM) -> bool:
+    return llm.model in CHAT_MODELS
```

Comment:
Looks like this broke quite a few cohere tests -- rather than checking llm.model, maybe an `isinstance()` check is better? 

Something like

```
def is_cohere_model(llm: BaseLLM) -> bool:
  from llama_index.llms.cohere import Cohere
  return isinstance(llm, Cohere)
```